Analyzer/interface/CommonFunction.h

```diff
@@ -1505,23 +1505,17 @@ reco::DeDxData computedEdx(const int& run_number,
                            float* scaleFactors,
                            TH3* templateHisto = nullptr,
                            bool usePixel = false,
+                           bool useStrip = true,
                            bool useClusterCleaning = true,
-                           bool reverseProb = false,
                            bool useTruncated = false,
-                           std::unordered_map<unsigned int, float>* TrackerGains = nullptr,
-                           bool useStrip = true,
                            bool mustBeInside = false,
                            size_t MaxStripNOM = 999,
                            bool correctFEDSat = false,
                            int crossTalkInvAlgo = 0,
                            float dropLowerDeDxValue = 0.0,
-                           dedxHIPEmulator* hipEmulator = nullptr,
```

Comment:
same story, current code doesnt use it, and anyway we dont need this for UL

Reply:
Same thing than previously, it's useful for the first runs of 2016.